<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_UserSettingMenu</class>
 <widget class="QWidget" name="FORM_UserSettingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>用户设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/909.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>998</x>
     <y>150</y>
     <width>285</width>
     <height>500</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="menu_3">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/ico_260/时间日期设置.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="menu_1">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Zen Hei</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
      </property>
      <property name="text">
       <string>时间日期设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="menu_4">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/ico_260/亮度设置.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="menu_2">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Zen Hei</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
      </property>
      <property name="text">
       <string>亮度设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="menu_6">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/ico_260/语言设置.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="menu_5">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Zen Hei</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
      </property>
      <property name="text">
       <string>语言设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="menu_8">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/ico_260/系统初始化.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="menu_7">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Zen Hei</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
      </property>
      <property name="text">
       <string>程序升级</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
